integration_tests/ctesque/AndroidManifest.xml

```diff
@@ -8,9 +8,6 @@
       android:minSdkVersion="16"
       android:targetSdkVersion="27" />
 
-  <!-- For SettingsTest -->
-  <uses-permission android:name="android.permission.WRITE_SECURE_SETTINGS" />
```

Comment:
After moved to `androidTest`, the `SettingsTest` under `ctesque` can't work correctly with required permission `android.permission.WRITE_SECURE_SETTINGS`. I have tried to assign it with [`GrantPermissionRule`](https://developer.android.com/reference/androidx/test/rule/GrantPermissionRule), but it didn't work because `WRITE_SECURE_SETTINGS` is a critical permission, [not allowed for normal thrid-party apps](https://developer.android.com/reference/android/Manifest.permission#WRITE_SECURE_SETTINGS). So I moved `SettingsTest` to `ShadowSettingsTest`, and it only run with `Robolectric` now. After that,  `android.permission.WRITE_SECURE_SETTINGS` is not needed anymore, because it was added for `SettingsTest`.

Reply:
I think [instrumentation test in AOSP](https://source.android.com/compatibility/tests/development/instrumentation) can grant this permission successfully, but it is not easy for normal app tests, in my opinion. I keep this change part, and resolve this conversation.